src/calls/get-equipment.ts

```diff
@@ -5,24 +5,24 @@ export interface GetEquipmentInput {
 }
 const operatingHoursSchema = z.object({
   hours: z.number(),
-  unit_driven: z.boolean(),
+  unitDriven: z.boolean(),
 });
```

Comment:
We need to transform the data after parsing, like so: 

```suggestion
  unit_driven: z.boolean(),
}).transform(({unit_driven}) => ({unitDriven: unit_driven}));
```

This applies to every zod schema.